src/lib/split-view/split-view-panel/split-view-panel.ts

```diff
@@ -0,0 +1,214 @@
+import { CustomElement, attachShadowTemplate, FoundationProperty, coerceBoolean, coerceNumber } from '@tylertech/forge-core';
+import { tylIconDragVerticalVariant } from '@tylertech/tyler-icons/extended';
+import { tylIconDragHandle } from '@tylertech/tyler-icons/standard';
+
+import { BaseComponent, IBaseComponent } from '../../core/base/base-component';
+import { SplitViewPanelPosition, SPLIT_VIEW_PANEL_CONSTANTS } from './split-view-panel-constants';
+import { SplitViewPanelFoundation } from './split-view-panel-foundation';
+import { SplitViewPanelAdapter } from './split-view-panel-adapter';
+import { SplitViewOrientation } from '../split-view/split-view-constants';
+import { ISplitViewBase } from '../core/split-view-base';
+import { IconComponent, IconRegistry } from '../../icon';
+import { RippleComponent } from '../../ripple';
+
+import template from './split-view-panel.html';
+import styles from './split-view-panel.scss';
+
+export interface ISplitViewPanelComponent extends ISplitViewBase, IBaseComponent {
+  position: SplitViewPanelPosition;
+  size: number | string;
+  min: number;
+  max: number | undefined;
+  label: string;
```

Comment:
Let's rename this to something like `accessibleLabel`/`accessible-label` to make it more clear.